src/main/java/com/marklogic/contentpump/AggregateXMLReader.java

```diff
@@ -217,6 +217,9 @@ protected void removeNameSpaceDecl() {
     private void processStartElement() throws XMLStreamException {
         String name = xmlSR.getLocalName();
         String namespace = xmlSR.getNamespaceURI();
+        if ("".equals(namespace)) {
```

Comment:
One suggestion I have here is to move this if after the LOG section to make the code more readable.